war/src/main/java/io/lumeer/engine/controller/ConfigurationFacade.java

```diff
@@ -42,14 +42,14 @@
 public class ConfigurationFacade implements Serializable {
 
    public enum ConfigurationLevel {
-      USER, PROJECT, ORGANISATION
+      USER, PROJECT, ORGANIZATION, GLOBAL
```

Comment:
This hierarchy is kind of weird. Why do not we have all five levels here?

- USER_GLOBAL
- USER_PROJECT
- USER_ORGANIZATION
- PROJECT
- ORGANIZATION